python/cudf/dataframe/dataframe.py

```diff
@@ -2719,6 +2734,16 @@ def _apply_support_method(self, method, **kwargs):
             result = result.set_index(self._cols.keys())
         return result
 
+    def _columns_view(self, columns):
+        """
+        Return a subset of the DataFrame's columns as a view.
+        """
+        columns = as_index(columns)
+        result_columns = OrderedDict({})
```

Comment:
Do we need to go down to the OrderDict level? Is there not a way to call a dataframe `__setitem__` by reference?

Reply:
Unfortunately it looks like `._cols` (OrderedDict) is the only attribute of a DataFrame that exposes it as a view